multiqc/modules/peddy/peddy.py

```diff
@@ -205,3 +212,31 @@ def peddy_relatedness_plot(self):
                 <span style="color: #2B9F2B;">greather than 0.5</span>.""",
                 plot = scatter.plot(data, pconfig)
             )
+            
+    def peddy_het_check_plot(self):
+        """plot the het_check scatter plot"""
+        # empty dictionary to add sample names, and dictionary of values
+        data = {}
+        # for each sample, and list in self.peddy_data
+        for s_name, d in self.peddy_data.items():
+            # check the sample contains the required columns
+            if 'median_depth_het_check' in d and 'het_ratio_het_check' in d:
+                # add sample to dictionary with value as a dictionary of points to plot
+                data[s_name] = {
+                    'x': d['median_depth_het_check'],
+                    'y': d['het_ratio_het_check']
+                }
+        
+        pconfig = {
+            'id': 'peddy_het_check_plot',
+            'title': 'Peddy Het Check Plot',
+            'xlab': 'median depth',
+            'ylab': 'proportion het calls',
```

Comment:
Is it worth forcing a y axis scale from 0-1 here? Or would that mask differences that we're looking for?

Reply:
Yea, I think 0-1 Y axis would make the differences harder to spot. I have set the y limits to +/- 0.1 from min/max values, but now I think about it I prefer it with the original padding! Will undo this change